data/os/RedHat.yaml

```diff
@@ -0,0 +1,35 @@
+---
+
+profile_acctd::required_pkgs:
+  - "perl-Config-Simple"
+  - "perl-LDAP"
+  - "perl-Sys-Syslog"  # also used by profile_audit::root_equivalence.pp
+  - "perl-Text-Unidecode"
+  - "perl-JSON"
+  - "perl-DBI"
+  - "perl-DBD-mysql"
+  - "perl-MailTools"
+  - "perl-LWP-Protocol-https"
+profile_acctd::crons: {}
```

Comment:
It looks like this hiera value and the examples could be put in the regular `data/common.yaml` instead of in the RedHat specific one

Reply:
Indeed, adjusted!